src/triage/experiments/base.py

```diff
@@ -65,6 +67,7 @@ class ExperimentBase(ABC):
         project_path (string)
         replace (bool)
         cleanup_timeout (int)
+        profile (bool)
```

Comment:
More bikeshedding, arguably -- but, we could make this argument default to `None`, and accept instead an absolute or relative path.

Granted, this _might_ not be useful; but, with this sort of thing, it's a nice way to offer that configurability, rather than choosing a set directory path and name (profiling_stats).

Moreover, the boolean logic could remain the same. Only in the `get_store` call could we make use of the setting, _e.g._:

```
store = self.project_storage.get_store([self.profile], …)
```

Or I suppose if we _wanted_ to support arbitrary absolute paths:

```
if os.path.isabs(self.profile):
    store = self.project_storage.storage_class(self.profile, filename)
else:
    store = self.project_storage.get_store([self.profile], filename)
```

Reply:
I'd rather start with convention over configuration unless people specifically ask for the configuration in projects. This is the simplest possible interface, I think, and that's what I'd prefer to start with.

Reply:
Works for me